parlai/agents/image_seq2seq/image_seq2seq.py

```diff
@@ -120,13 +120,16 @@ def batchify_image_features(self, batch: Batch) -> Batch:
 
         Image features represented by tensors will set to the right type.
         """
-        if type(batch.image) == list and any(b is not None for b in batch):
+        if type(batch.image) == list and any(b is not None for b in batch.image):
             images = []
             for img in batch.image:
                 if isinstance(img, torch.Tensor):
                     img = self._process_image_features(img)
                 images.append(img)
             batch.image = images
+        else:
+            images = [None for _ in range(len(batch.valid_indices))]
```

Comment:
nit: this can also be `[None] * len(batch.valid_indices)`, and similarly for below